ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -97,6 +97,7 @@ private[platform] object CompletionFromTransaction {
       status = maybeStatus,
       transactionId = transactionId,
       applicationId = applicationId,
+      submissionId = maybeSubmissionId.getOrElse(""),
```

Comment:
Isn't this filled in the next lines?

Reply:
You're right! Let me leave the test, then.